src/BizHawk.Emulation.Cores/CPUs/MOS 6502X/MOS6502X.cs

```diff
@@ -245,6 +245,18 @@ public bool FlagN
 			private set => P = (byte)((P & ~0x80) | (value ? 0x80 : 0x00));
 		}
 
+		/// <summary>
+		/// For the unsupported opcode, ANE.
+		/// If your core requires a specific constant for this opcode, set it here.
+		/// </summary>
+		public Func<byte> AneConstantFunc { get; set; }

```

Comment:
If these are per-console constants, do they need to be delegates?

Reply:
Hmm, perhaps not - some docs say that these can vary as a system is running, so it was implemented in a way that this could be accomplished. Some docs call them "constants" but it's not a great name, it's said to be a kind of entropic value. However, it is probably safe to assume for a "pristine" emulator state that these can stay constant. They don't seem to vary in any of the real hardware I have, for what it's worth.

Reply:
Resolved in commit 3553760